install/kubernetes/cilium/templates/clustermesh-apiserver/tls-cronjob/_job-spec.tpl

```diff
@@ -19,7 +19,7 @@ spec:
             {{- if .Values.debug.enabled }}
             - "--debug"
             {{- end }}
-            - "--ca-generate"
+            - "--ca-generate={{ .Values.certgen.generateCA }}"
```

Comment:
The same change shall be applied to `install/kubernetes/cilium/templates/hubble/tls-cronjob/_job-spec.tpl` as well.